### PR TITLE
Consistently show merch cost in conversation

### PR DESCRIPTION
**Pull request by haveric**
#### Summary
Bugfixes "Consistently show merch cost in conversation"

#### Purpose of change

Some purchases using merch weren't showing the price. The Tacoma ranch barber was in $ and the scavenger was [FMC750]. We should be consistent for all of them. Also fixed the grammar in Cody's blacksmith quests.

#### Describe the solution

Anywhere merch is used to purchase/trade for something, show `[# merch]` before the sentence for consistency. Only exception now is Cody's fabrication services, which are shown as `[# merch, # mm]`

#### Describe alternatives you've considered

- Maybe this could be a variable or automatically handled by looking at `u_sell_item` and `u_consume_item`, but I'm not sure we want to do this for all items.

#### Testing

Compiled and loaded the game. Checked a couple dialogs.

To find these, I searched for `"u_consume_item": "FMCNote"` and `"u_sell_item": "FMCNote"`

#### Additional context

image
image

### CODE REVIEW

File: data/json/npcs/tacoma_ranch/NPC_ranch_barber.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
       { "text": "What is your job here?", "topic": "TALK_RANCH_BARBER_JOB" },
       { "text": "Do you need any help?", "topic": "TALK_RANCH_BARBER_HIRE" },
       {
-        "text": "[$5] I'll have a shave",
+        "text": "[3 merch] I'll have a shave",
```
Comment (Faalagorn) [inline data/json/npcs/tacoma_ranch/NPC_ranch_barber.json:21]: Missing full stop at the end.

File: data/json/npcs/tacoma_ranch/NPC_ranch_barber.json [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
         "topic": "TALK_RANCH_BARBER_CUT",
         "condition": { "u_has_items": { "item": "FMCNote", "count": 3 } },
         "effect": [ "buy_shave", { "u_sell_item": "FMCNote", "count": 3 }, "barber_beard" ]
       },
       {
-        "text": "[$10] I'll get a haircut",
+        "text": "[6 merch] I'll get a haircut",
```
Comment (Faalagorn) [inline data/json/npcs/tacoma_ranch/NPC_ranch_barber.json:27]: Missing full stop at the end.

**Comment by Faalagorn** [comment]
I marked two comments that are missing the full stop at the end (these are on screenshot), these cause the strings to appear (and having to be translated) twice in Transifex, that's how I found them :)

**Comment by haveric** [comment]
I can make those fixes in the Tacoma PR

**Pull request merged**